CUETools.Codecs.FlaCuda/FlaCudaWriter.cs

```diff
@@ -206,7 +206,7 @@ public int CompressionLevel
 				if (value < 0 || value > 11)
 					throw new Exception("unsupported compression level");
 				_compressionLevel = value;
-				eparams.flake_set_defaults(_compressionLevel, !_settings.GPUOnly);
+				eparams.Flake_set_defaults(_compressionLevel, !_settings.GPUOnly);

```

Comment:
Please undo. The lower case letter at the beginning of the method name should stay, like in other methods.